api-tests/secure-pipeline/run-tests.sh

```diff
@@ -19,23 +29,48 @@ export CRI_STUB_GEN_CRED_API_KEY
 MANAGEMENT_TICF_API_KEY=$(aws secretsmanager get-secret-value --secret-id /build/core/credentialIssuers/ticf/connections/stub/apiKey | jq -r .SecretString)
 export MANAGEMENT_TICF_API_KEY
 
-MANAGEMENT_CIMIT_STUB_API_KEY=$(aws ssm get-parameter --name /tests/core-back-build/cimit_api_key | jq -r .Parameter.Value)
+MANAGEMENT_CIMIT_STUB_API_KEY=$(aws secretsmanager get-secret-value --secret-id /build/core/cimitManagementApi/apiKey | jq -r .SecretString)
```

Comment:
this secret has been created in build. I'm not sure why we're pulling this secret specifically from ssm rather than secrets manager, but better to be consistant. also the traffic codebuild doesn't have perms to pull this ssm param